### ISSUE TITLE
speechToTextWF: notify speech_to_text_generation_service that there is content to be STTed

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/common-accessioning#1354]
blocked by https://github.com/sul-dlss/speech-to-text/issues/2, which should probably be decided before this is started?

options being considered (feel free to add others for consideration if you think of something better)
- put a message on an AWS SQS queue that is watched by our cloud [speech_to_text_generation_service](https://github.com/sul-dlss/speech-to-text/issues/3) (or by [SageMaker](https://github.com/sul-dlss/speech-to-text/issues/4), if we go that route)
- call a REST API that we implement (create a separate ticket that blocks this one if we go that route)

Another option we considered was having our speech_to_text_generation_service watch an S3 bucket for incoming content, which @edsu was able to prototype successfully.  but after discussion this past fri 2024-09-20, we agreed that explicitly passing messages between services would likely be more reliable and intelligible, and less locked to AWS' specific S3 implementation.

**Comment by jmartin-sul** [comment]
to use for updating this or the possible dupe ticket after planning meeting:
- we think we settled on SQS
- there may be a WIP PR that does most or all of this: https://github.com/sul-dlss/common-accessioning/pull/1367

see also #1356, this may be a dupe of that?  close one if so?

**Comment by jmartin-sul** [comment]
closing as a duplicate of #1356
